scripts/auto-rebase/rebase_odf-lvm.sh

```diff
@@ -0,0 +1,144 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+log() {
+    local msg="${*}"
+    printf "\E[34m%s\E[00m\n" "${msg}"
+}
+
+install_yq() {
+    # shellcheck disable=SC2155
+    local os="$(uname -s | tr '[:upper:]' '[:lower:]')"
+    # shellcheck disable=SC2155
+    local arch="$(uname -m)"
+    if [ "$arch" == "x86_64" ]; then
+        arch="amd64"
+    fi
+    local package="yq_${os}_${arch}"
+
+    tmpdir=$(mktemp -d)
+
+    pushd "$tmpdir"
+    wget https://github.com/mikefarah/yq/releases/v4.29.2/download/{'extract-checksum.sh','checksums_hashes_order','checksums',"${package}"}
+    chmod +x ./extract-checksum.sh
+    diff <(./extract-checksum.sh SHA-256 "$package" | awk '{print $2"  "$1}') <(sha256sum "$package") || \
+        { echo >&2 "Checksum validation failed"; return 1;}
+    chmod u+x "./$package"
+    mv "$package" /usr/local/bin/yq
+    popd
+}
+
+fetch_operator_bundle() {
+    local image_digest=$1
+    local arch=$2
+
+    mkdir -p "${ODF_STAGING}/${arch}/bundle"
+    pushd "${ODF_STAGING}/${arch}/bundle"
+    oc image extract  --path /manifests/:. --filter-by-os "$arch" "$image_digest" || \
+        { popd; return 1 ;}
+    popd
+    echo "$image_digest"
+}
+
+parse_images() {
+    local src=$1
+    yq '.spec.relatedImages[]? | [.name, .image] | @tsv' "${src}/bundle/${CSV_FILE}" > "${src}/images"
+    echo "${src}/images"
+}
+
+update_release_go() {
+    set -x
+    local src=$1
+    local arch=${2:-} # allow null arch input, assume release.go
+
+    case "$arch" in
+    amd64)
+        release_file="${REPOROOT}/pkg/release/release_amd64.go"
+        ;;
+    arm64)
+        release_file="${REPOROOT}/pkg/release/release_arm64.go"
+        ;;
+    *)
+        release_file="${REPOROOT}/pkg/release/release.go"
+        ;;
+    esac
+
+    local staged_release
+    staged_release="$(mktemp -d)/$(basename "$release_file")"
+    cp "$release_file" "${staged_release}"
+    # shellcheck disable=SC2155
+    local images="$(parse_images "$src")"
+    # ensure that if a sed substitute fails, we bail out rather than clobber a release file
+    if ! (
```

Comment:
We go to a lot of trouble to parse and update the release.go file here and in the main rebase script. What if we wrote a YAML file to the assets directory instead, and had release.go load and parse it? We would want to make the same change for all images, so we should not hold up this PR over the discussion.

Reply:
Yeah, I think you're right.  We go through a lot of trouble to do hacky string replacements in the release*.go files.  We'd probably be better off reusing our bindata pipeline to embed the release image data in the binary from a yaml source, rather than all the shell-driven string substitutions we do now.

Reply:
In general, though, I agree this script should align more with the other rebase script in style and reuse code.